tools/codeql/pipeline.py

```
@@ -0,0 +1,108 @@
+                    f"SARIF normalization failed: {exc}"
+                )
+                manifest["status"] = "failed" if config.fail_policy == "hard" else "soft-failed"
+
+    # Step 4: import risk
+    signals_path = normalized_dir / "file-signals.yml"
+    risk_path = ROOT / "itemdb/notes/file-risk-index.yml"
+    if signals_path.is_file():
+        try:
+            import_risk(signals_path, risk_path)
+            _progress(progress, "CodeQL: imported file risk signals")
+        except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)